scripts/modSmCh.mac

```diff
@@ -1,8 +1,17 @@
 # Macro setting barrel and modular detector (24 part ring)
 # Using small annihilation chamber
 
+/jpetmc/detector/loadTargetForRun 5
+
+# Loading standard scintillators (3 layers)
+/jpetmc/detector/loadJPetBasicGeom
+
+# Loading modular layers
 /jpetmc/detector/loadModularLayer Single
-/jpetmc/detector/loadGeomForRun 5
+
+# Loading scintillators without frame
+/jpetmc/detector/loadOnlyScintillators
+
```

Comment:
Here it would be proper to add new option or writing out json file for modular configuraion 
`/jpetmc/detector/createGeometryFile`

Reply:
I added flags with some comment what they do